WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -418,6 +418,28 @@ extension ZendeskUtils {
             createRequest()
         }
     }
+
+    /// Creates an identity for the current account without confirming the user
+    /// email or username.
+    ///
+    /// - warning: This method should only be used for interactions with Zendesk
+    /// that doesn't require a confirmed identity and/or a response from support,
+    /// such as sending feedback about the app.
+    @MainActor func createIdentifitySilently() async {
+        // If we already have an identity, and the user has confirmed it, do nothing.
+        let haveUserInfo = ZendeskUtils.sharedInstance.haveUserIdentity && ZendeskUtils.sharedInstance.userNameConfirmed
```

Comment:
It's a bit of duplication, but I'm not up to refactoring this whole class. The priority was to not break any of the existing flows.